CMakeLists.txt

```diff
@@ -258,6 +258,9 @@ if (NOT TARANTOOL_VERSION)
     message (FATAL_ERROR "Unable to retrive version from git or ${VERSION_FILE} file.")
 endif()
 
+set(TARANTOOL_VERSION_SUFFIX "" CACHE STRING "Version suffix.")
+set(TARANTOOL_VERSION "${TARANTOOL_VERSION}${TARANTOOL_VERSION_SUFFIX}")
```

Comment:
Should not we write it into the VERSION file as well?

Reply:
I removed this patch. Instead now, I disable any version detection if `TARANTOOL_VERSION` is already set. A good thing about it is that we can use the new Tarantool EE repository instead of the legacy one in tarantool/sdk almost without any changes to sdk CMake configuration files.

Re the VERSION file, I think we don't need to generate it if `TARANTOOL_VERSION` is set. At least, the legacy EE CMake config doesn't do it. If the user overrides `TARANTOOL_VERSION`, they should know what they're doing and write the version to the VERSION file by themselves if they need to.